engine/src/test/scala/cromwell/webservice/CromwellApiServiceSpec.scala

```diff
@@ -589,36 +468,55 @@ class CromwellApiServiceSpec extends FlatSpec with CromwellApiService with Scala
       }
   }
 
-  ignore should "return good results for a good query" in {
-    Get(s"/workflows/$version/query?status=Succeeded") ~>
+  it should "return good results for a good query" in {
+    val workflowId = WorkflowId.randomId()
+    val runningId = WorkflowId.randomId()
+    val events = Seq(
+      MetadataEvent(MetadataKey(workflowId, None, WorkflowMetadataKeys.Status), MetadataValue("Succeeded")),
+      MetadataEvent(MetadataKey(runningId, None, WorkflowMetadataKeys.Status), MetadataValue("Running"))
+    )
+
+    publishMetadata(events)
+    forceSummary()
```

Comment:
ToL: At this point, are we really testing the API, or testing the underlying `MetadataService` architecture? I ask because we're now requiring a lot of knowledge of its inner workings in a logically unrelated test

It feels like to be intellectually honest we should mock out the `ServiceRegistryActor` to provide what we want to the REST API, and check that the REST API is acting correctly in various conditions.